data/tutorials/lg_04_labels.md

```diff
@@ -45,7 +54,7 @@ val range : first:int -> last:int -> int list = <fun>
 ```
 
 (Notice that both `to` and `end` are reserved words in OCaml, so they
-cannot be used as labels. So you cannot have `~from/~to` or
+cannot be used as labels. This means you cannot have `~from/~to` or
```

Comment:
This is confusing (no issue for `~from` and `~start`), you just cannot have `~to` and `~end` but you can have `~to_` and `~end_` for example.

Reply:
Thank you for your comment, @gpetiot. Shall I change it to `~from/~to_`, etc?

Reply:
The example is fine with `~first` and `~last`. I would just fix the note: "This means you cannot have `~to` or `~end`".

Reply:
Changed! Please double check I did it correctly.

Reply:
All good!